src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -5,20 +5,21 @@
 
 #include <base/hex.h>
 #include <base/interpolate.h>
+#include <Common/logger_useful.h>
+#include <Common/FailPoint.h>
 #include <Common/Macros.h>
 #include <Common/MemoryTracker.h>
 #include <Common/ProfileEventsScope.h>
 #include <Common/StringUtils.h>
+#include <Common/ThreadFuzzer.h>
 #include <Common/ZooKeeper/KeeperException.h>
 #include <Common/ZooKeeper/Types.h>
 #include <Common/escapeForFileName.h>
 #include <Common/formatReadable.h>
 #include <Common/noexcept_scope.h>
+#include <Common/randomDelay.h>
```

Comment:
```suggestion
#include <Common/logger_useful.h>
#include <Common/Macros.h>
#include <Common/MemoryTracker.h>
#include <Common/ProfileEventsScope.h>
#include <Common/StringUtils.h>
#include <Common/ZooKeeper/KeeperException.h>
#include <Common/ZooKeeper/Types.h>
#include <Common/escapeForFileName.h>
#include <Common/formatReadable.h>
#include <Common/noexcept_scope.h>
```

Reply:
We cann't remove FailPoint and ThreadFuzzer.